packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -172,9 +186,9 @@ export default class SelectOpener extends React.Component<
                 /* Note(marcysutton): type=button prevents form submits on click */
                 type="button"
                 style={style}
-                onClick={!disabled ? this.handleClick : undefined}
-                onKeyDown={!disabled ? this.handleKeyDown : undefined}
-                onKeyUp={!disabled ? this.handleKeyUp : undefined}
+                onClick={allowInteraction ? this.handleClick : undefined}
+                onKeyDown={allowInteraction ? this.handleKeyDown : undefined}
```

Comment:
I'm still able to open the SingleSelect and MultiSelect with the down arrow key in all browsers. Here is a screenshot:

<img width="639" height="739" alt="Opened SingleSelect" src="https://github.com/user-attachments/assets/f2f1437f-76b5-4275-909c-bb7c78f9ff2a" />

I believe the logic for arrow key handling is in DropdownCore?